api/Request.json

```diff
@@ -902,75 +692,31 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Request/headers",
           "support": {
-            "chrome": [
-              {
-                "version_added": "42"
-              },
-              {
-                "version_added": "41",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "Experimental Web Platform Features"
-                  }
-                ]
-              }
-            ],
+            "chrome": {
+              "version_added": "42"
```

Comment:
Was this really removed from Chrome for Android but continues to ship in Chrome desktop?

Reply:
Fixed up the data in #10070!